channels/packages/after-install/0.1.1/pod-disruption-budget.yaml

```diff
@@ -3,7 +3,7 @@ kind: PodDisruptionBudget
 metadata:
   name: storageos-node-guard
 spec:
-  maxUnavailable: 1
+  minAvailable: 99999
```

Comment:
Does it work to specify this as `"100%"` instead? as it says in the api https://github.com/kubernetes/api/blob/a26a16a095cab454e928a95c533b8cf1b80aa2ec/policy/v1/types.go#L31

Reply:
@nolancon after install set the right number before creation, so it doesn't matter